packages/paste-website/src/pages/utilities/screen-reader-only/index.mdx

```diff
@@ -0,0 +1,212 @@
+
+<LivePreview scope={{Card, ScreenReaderOnly, Heading, Paragraph, Button, Anchor, Flex}} language="jsx">
+  {`<Card>
+  <ScreenReaderOnly as="div">
+    <Heading as="h2" variant="heading20">Flex</Heading>
+  </ScreenReaderOnly>
+  <Paragraph>
+    Deploy an omnichannel contact center within minutes and customize every element of the experience, including the agent desktop, channels, interaction routing, and reporting using common web frameworks like React.js and REST APIs.
+  </Paragraph>
+  <Flex vAlignContent="center">
+    <Flex marginRight="space40">
+      <Button variant="primary">
+        Launch Flex
+      </Button>
+    </Flex>
+    <Anchor href="#">
+      View all projects
+    </Anchor>
+  </Flex>
+</Card>`}
+</LivePreview>
+
+### Repetitive buttons and hidden headers
+
+Each column in a table is required to have a column header. Sometimes those column headers can be visually hidden from sighted users if desired. To do so use the Screen Reader Only component to hide the cell content for that column header.
+
+Another common use case is for providing context to repetitive actions. Visually the "Buy" button is associated with the phone number via the row highlighting, but to a screen reader user the "Buy" action might be unclear. To provide additional clarification we can add _what_ you would be buying as visually hidden text to the button.
+
+<LivePreview scope={{Table, Thead, Tbody, Tr, Td, Th, Button, Text, ScreenReaderOnly}} language="jsx">
+  {`<Table>
+  <Thead>
+    <Tr>
+      <Th>Number</Th>
+      <Th>Type</Th>
+      <Th>Monthly Fee</Th>
+      <Th>
+        <ScreenReaderOnly>Actions</ScreenReaderOnly>
+      </Th>
+    </Tr>
+  </Thead>
+  <Tbody>
+    <Tr>
+      <Th>
+        <strong>+1 (903) 603-9469</strong>
+        <Text as="div" textColor="colorTextWeak" fontWeight="normal">
+          Mabank, TX
+        </Text>
+      </Th>
+      <Td>local</Td>
+      <Td>$1.00</Td>
+      <Td>
+        <Button variant="secondary" size="small">
+          Buy <ScreenReaderOnly>+1 (903) 603-9469</ScreenReaderOnly>
+        </Button>
+      </Td>
+    </Tr>
+    <Tr>
+      <Th>
+        <strong>+1 (470) 309-0026</strong>
+        <Text as="div" textColor="colorTextWeak" fontWeight="normal">
+          Adairsville, GA
+        </Text>
+      </Th>
+      <Td>local</Td>
+      <Td>$1.00</Td>
+      <Td>
+        <Button variant="secondary" size="small">
+          Buy <ScreenReaderOnly>+1 (470) 309-0026</ScreenReaderOnly>
+        </Button>
+      </Td>
+    </Tr>
+    <Tr>
+      <Th>
+        <strong>+1 (484) 662-4088</strong>
+        <Text as="div" textColor="colorTextWeak" fontWeight="normal">
+          Hamburg, PA
+        </Text>
+      </Th>
+      <Td>local</Td>
+      <Td>$1.00</Td>
+      <Td>
+        <Button variant="secondary" size="small">
+          Buy <ScreenReaderOnly>+1 (484) 662-4088</ScreenReaderOnly>
+        </Button>
+      </Td>
+    </Tr>
+  </Tbody>
+</Table>`}
+</LivePreview>
```

Comment:
I know this is a useful pattern but am struggling to know _when_ to use it more examples would definitely help

Reply:
A little research uncovered a few patterns 
https://webaim.org/techniques/css/invisiblecontent/#skipnavlinks

```html
<div class=”screen-reader-only”>These are screen reader user specific instructions.</div>
<div aria-hidden=”true”>These are instructions for everyone else.</div>
```

https://accessible360.com/accessible360-blog/use-aria-label-screen-reader-text/

Reply:
^ this is great, we should potentially add this

Reply:
Yeah this one is a bit hard. I was scouring sendgrid and console for examples I could rebuild with the stuff we have in Paste at the moment and this is the best I could do. It's like a chicken and the egg situation